examples/with_cognito_authentication/main.tf

```diff
@@ -18,12 +18,11 @@ locals {
 }
 
 module "subnets" {
-  source               = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=tags/0.19.0"
+  source               = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=tags/0.26.0"
   availability_zones   = local.availability_zones
   namespace            = var.namespace
   stage                = var.stage
   name                 = var.name
-  region               = var.region
```

Comment:
I think the region needs to stay

Reply:
I removed it because `terraform validate` said:

```
Error: Unsupported argument

  on main.tf line 26, in module "subnets":
  26:   region                   = var.region

An argument named "region" is not expected here.
```